### ISSUE TITLE
How to create sms gateway for drupal

### ISSUE THREAD

**Issue by ugintl** [mikehchase/drupal7-sms-test#2]
I have found some open source gateways which I would like to use with drupal as sms gateway and alternative to twilio etc. I would use it with sms framework module.

Can you guide me how I can do it?

Here you will find open source sms gateways. https://socialcompare.com/en/comparison/sms-gateways-1odvv6xv

**Comment by mikehchase** [comment]
No, I'm sorry. The module is pretty easy to code to, so you should be able
to get that going on your own, provided you have some PHP & PHP DB and
Drupal 7 custom module background. This code is out of date and is getting
deleted.
Michael H Chase
